docs/product/stats/index.mdx

```diff
@@ -23,9 +14,82 @@ With the dropdowns at the top of the page, you can set whether the page displays
   event type.
 </Note>
 
+
+There are three tabs available on the page: [Usage](#usage-stats), [Issues](#issue-stats), and [Health](#health-stats).
+
+The "Usage" tab shows your organization’s event and attachments usage, while also providing a high-level overview of how different projects are using Sentry. The "Issues" tab shows a breakdown of the issues assigned to your team and how they triaged them, while the "Health" tab provides an overview of your team’s project health.
+
+
+## Usage Stats
+
+The "Usage" tab provides an overview of the events (errors and transactions) and attachments that Sentry has received across your entire organization. It includes a detailed breakdown of each project's consumption categorized as _Accepted_, _Filtered_, _Rate Limited_, _Invalid_ or _Client Discard_.
+
+### Accepted
+
+Events and attachments that were successfully processed and stored.
+
+### Filtered
+
+Filtered events and attachments in Sentry are intentionally excluded based on predefined criteria to reduce noise and prioritize relevant data. The following reasons are currently defined:
+
+- **Browser Extensions**: Filtered by browser extension.
+- **Error Message**: Filtered based on specific error messages.
+- **Filtered Transactions**: Filtered because it was a call to a filtered transaction.
+- **Invalid CSP**: Filtered due to an invalid Content Security Policy (CSP) policy.
+- **Ip Address**: Filtered by IP address.
+- **Legacy Browsers**: Filtered by certain legacy versions of browsers that are known to cause problems.
+- **Localhost**: Filtered due to localhost restriction.
+- **Others**: Filtered due to other generic filtering rules not specified above.
+- **Release Version**: Filtered by release name (version).
+- **Web Crawlers**: Identified as a known web crawler.
+
+For more details, please consult the [Inbound Filters](/concepts/data-management/filtering/) documentation.
+
+### Rate Limited
+Events and attachments discarded due to rate limits or quota. The following reasons are currently defined:
+
+- **Abuse**: Activated when Sentry is misused.
+- **Cardinality Limited**: Exceeded internally defined cardinality limit.
+- **Rate Limited**: Exceeded allowed event rate.
+- **Spike Protection**: Activated to protect from any sudden quota-depleting spike. See [Spike Protection](/pricing/quotas/spike-protection/) for more information.
```

Comment:
@jan-auer what is your opinion on this list? I might remove "Rate Limited" as it can be confusing

Reply:
Here's a suggestion for how to structure rate limits. Please expand this with links to relevant documentation as appropriate:

```md
- **Quota**: The monthly quota of your subscription was depleted and there is no on-demand budget left.
- **Key Limit**: Traffic exceeded a rate limit defined on the client key (DSN).
- **Spike Protection**: Activated to protect from a sudden spike in data volume. See [Spike Protection](/pricing/quotas/spike-protection/) for more information.
- **Internal Limit**: A rate limit for excessive volume was enforced by Sentry. These limits are not configurable.
```

Reason codes for the above are:
- quota: `*_usage_exceeded` (everything that ends with `"_usage_exceeded"`)
- key limit: `key_quota`
- spike protection: `smart_rate_limit`
- internal: _(everything else)_

Therefore, please include the "abuse" and "cardinality" limit outcomes under "Internal Limits" for now.